lib/AppInfo/Application.php

```diff
@@ -37,5 +40,14 @@ class Application extends App {
 	 */
 	public function __construct(array $params=[]) {
 		parent::__construct('calendar', $params);
+
+		$container = $this->getContainer();
+
+		// TODO: Migrate to new bootstrap once Calendar supports only Nextcloud 20+
```

Comment:
We can just bump to 2.1 and limit it to Nc 20+

Reply:
Sounds good to me :+1:

Reply:
@tcitworld Do you have any objections here?
I would release this as part of Calendar 2.1 and we can always backport bugfixes to 2.0.x when necessary :)

Reply:
Suits me. There's at least #2286 and #2275 that would be nice to have in a close 2.0.x release (preferably released before 20 gets out with 2.1).